test/e2e-tests-standing-order.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+cd dex-contracts/
+
+# Place standing order in current Auction (accountId, buyToken, sellToken, minBuy, maxSell)
+truffle exec scripts/standing_order.js 0 1 2 12 12
```

Comment:
I didn't realize we had these scripts already! or is this a preemptive change?

Reply:
https://github.com/gnosis/dex-contracts/pull/133